File: CapacitacionCM/Practicas/Python/1-Inversion.py



cantidad = float(input("Ingresar la candad a invertir: "))
interes_anual = float(input("Ingresar interes anual en %: "))
anos = float(input("Ingresar el numero de anos de la inversion: "))

capital = cantidad *(1 + interes_anual / 100) ** anos

print("EL capital obtenido en la inversion es: ",round(capital,2))

File: CapacitacionCM/Practicas/JavaScript/1-Inversion.js
let cantidad = parseFloat(prompt("Ingresar la candad a invertir: "));
let interes_anual = parseFloat(prompt("Ingresar interes anual en %: "));
let anos = parseFloat(prompt("Ingresar el numero de anos de la inversion: "));

let capital = cantidad * Math.pow (1 + interes_anual / 100, anos);

console.log ("EL capital obtenido en la inversion es: " + capital.toFixed(2));